chainstory/linkapp/urls.py

```diff
@@ -1,6 +1,7 @@
-from django.urls import path
-from . import views
+from django.contrib import admin
+from django.urls import path, include
 
 urlpatterns = [
-    path('', views.lobby)
+    path('admin/', admin.site.urls),
+    path('', include('linkapp.urls')),
```

Comment:
What is the significance of "linkapp", is that just a name you chose for the project? Or is that some reserved keyword

Reply:
That was the name I chose for the first django app (not sure if I'll need another but I could). You have django projects and within them apps. I wanted this app to handle all communications between... everything. So linkapp seemed appropriate.